sources/platform/actors/running/actor_standby.md

```diff
@@ -0,0 +1,121 @@
+- **Desired requests per run** - The number of concurrent HTTP requests a single Standby Actor run is configured to handle. If this limit is exceeded, the system preemptively starts a new Actor run to handle the additional requests.
+- **Memory (MB)** - The amount of memory (RAM) allocated for the Actor in Standby mode, in megabytes. With more memory, the Actor can typically handle more requests in parallel, but this also increases the number of compute units consumed and the associated cost.
+- **Idle timeout (seconds)** - If a Standby Actor run doesn’t receive any HTTP requests within this time, the system will terminate the run. When a new request arrives, the system might need to start a new Standby Actor run to handle it, which can take a few seconds. A higher idle timeout improves responsiveness but increases costs, as the Actor remains active for a longer period.
+- **Build** - The Actor build that the runs of the Standby Actor will use. Can be either a build tag (e.g. `latest.`), or a build number (e.g. `0.1.2`).
+
+You can see these in the Standby tab of the Actor detail page. However, note that these properties are not configurable at the Actor level. If you wish to
+use the Actor-level hostname, this will always use the default configuration. To override this configuration, just create a new Task from the Actor.
+You can then head to the Standby tab of the created Task and modify the configuration as needed. Note that the task has a specific hostname, so make
+sure to use that in your application if you wish to use the custom configuration.
+
+#### Are the Standby runs billed differently
+
+No, the Standby runs are billed in the same fashion as the normal runs.
+However, running Actors in Standby mode might have unexpected costs, as the Actors run in the background and consume resources even when no requests are being sent until they are terminated after the idle timeout period.
+
+#### Are the Standby runs shared among users
+
+No, even if you use the Actor-level hostname with the default configuration, the background Actor runs for your requests are not shared with other users.
+
+#### How can I develop Actors using Standby mode
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You can head to the Settings tab of your Actor, enable Standby mode, and set the default configuration.
+![Standby for creators](./images/actor_standby/standby-creators.png)
+
+Actors using Standby mode must run a HTTP server listening on a specific port. The user requests will then be proxied to the HTTP server.
+You can get the port using the Actor configuration available in Apify SDK.
+
+If you need to override the port, you can do it via `ACTOR_STANDBY_PORT` environment variable.
+See example below with a simple Actor using Standby mode.
+
+<Tabs>
```

Comment:
This way the chosen language will be synced with other tab groups across the documentation
```suggestion
<Tabs groupId="main">
```